lib/tether/ops_linux.go

```diff
@@ -767,6 +798,34 @@ func (t *BaseOperations) MountLabel(ctx context.Context, label, target string) e
 	return nil
 }
 
+// mountDeviceLabel mounts
+func mountDeviceLabel(ctx context.Context, label string, target string) error {
+	// do..while ! timedout
+	var timeout bool
+	for timeout = false; !timeout; {
+		_, err := os.Stat(label)
+		if err == nil || !os.IsNotExist(err) {
+			break
+		}
+
+		deadline, ok := ctx.Deadline()
+		timeout = ok && time.Now().After(deadline)
```

Comment:
This pattern looks suspicious, why we are not simply waiting os.Stat and checking ctx.Done for timeout?